downstream/assemblies/platform/assembly-installing-controller-operator.adoc

```diff
@@ -33,8 +33,8 @@ When an instance of {ControllerName} is removed, the associated PVCs are not aut
 
 == Installing the {ControllerName} operator
 
-. Navigate to *Operators* > *Installed Operators*, then click on the *Ansible Automation Platform* operator.
-. Locate the *Automation controller* tab, then click *Create instance*.
+. Navigate to *Operators* > *Installed Operators*, then click on the btn:[Ansible Automation Platform] operator.
```

Comment:
This is actually a menu selection:

Navigate to menu:Operators[Installed Operators], then click the btn:[Ansible Automation Platform] operator.